persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/Cassandra311Persistence.java

```diff
@@ -202,6 +204,15 @@ protected void initializePersistence(Config config) {
       throw new RuntimeException("Unable to start Cassandra persistence layer", e);
     }
 
+    String hostId = System.getProperty("stargate.host_id");
+    if (hostId != null) {
```

Comment:
Not sure if empty String is likely to be passed but could skip both `null` and ""?